news/2020-02-04-community-contributors-2019.md

```diff
@@ -0,0 +1,206 @@
+
+Delving deep into the vagaries of leadership, **JBHyperion** is one of the extremely rare few who dared to stand up and take a swing at tackling the enormously complex social puzzle that is directing osu!'s modding scene. From his efforts creating the game's first social Code of Conduct for modders and mappers, to his intense dedication towards leading the NAT/QAT through some of the most tumultuous times the team has ever seen in osu!'s decade plus history, this one man has been at the forefront of it all, through good and bad.
+
+Largely unspoken and unsung by many, JBHyperion's efforts have dramatically reshaped the landscape of the osu!community in ways that we wager some will never truly be able to appreciate in full, but they are undoubtedly for the better, and it is safe to say that osu! has grown enormously from his time with us.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Mo.jpg)](https://osu.ppy.sh/users/2202163)
+
+Much like his brother in arms, **-Mo-** is another of the mighty few who dip their toes into the mire of leadership, spending hundreds of hours working with dozens of people in a concerted effort to improve the mapping and modding scene for everyone, travelling true throughout some of the team's rockiest years since their inception many, many years ago.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Chaos.jpg)](https://osu.ppy.sh/users/2628870)
+
+An organizer of the Community Mentorship Program and a long-term liaison between the osu!team and the Global Moderation Team, **Chaos** has been a stabilizing force in the community for many years now, with his efforts at home and abroad solidifying the future of the game. Beyond just this, he has also been heading the osu! Mapping Olympiad, a premier beatmapping contest series all the while.
+
+His work in managing the upkeep of the GMT and ensuring that the newest generations of osu!'s mappers and modders are guided through a labyrinthine set of customs and traditions, and helping them integrate into the game proper is essentially futureproofing the scene for years to come, whilst his efforts as a moderator ensure that the game's chat and forums remain safe for everyone.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/yaspo.jpg)](https://osu.ppy.sh/users/4945926)
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Uberzolik.jpg)](https://osu.ppy.sh/users/1314547)
+
+Essential to the continuation of the Community Mentorship Program, **yaspo and Uberzolik** have spent thousands of hours performing key tasks to ensure things stay on track.
+
+After Halfslashed stepped down, **yaspo** picked up the slack and stepped up to lead beside Okoratu, and has been responsible for running dozens of contests, hundreds of discussions, hand-picking similar numbers of mentor and mentee applications, hosting meetings and so much more.
```

Comment:
```suggestion
After Halfslashed stepped down, **yaspo** picked up the slack and stepped up to lead beside Okoratu, and has been responsible for running dozens of contests and discussions, hand-picking several hundreds of mentor and mentee applications, hosting meetings and so much more.
```

Discussions aren't nearly that many, but mentor and mentee applicantions certainly are